projects/max14906/src/examples/dummy/dummy_example.h

```diff
@@ -0,0 +1,13 @@
+#ifndef __DUMMY_EXAMPLE_H__
+#define __DUMMY_EXAMPLE_H__
+
+/** Defining specific mask for this example. */
+#define MAX14906_SLED_CH(x)	NO_OS_BIT(x + MAX14906_CHANNELS)
+
+static const char *climit_name[] = {
+	"600mA", "130mA", "300mA", "1.2A",
+};
```

Comment:
can be moved to .c file seeing how they are not used anywhere else

Reply:
Will move.